actions/npm-check/npm_check.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+
+// trunk-ignore-all(eslint)
```

Comment:
Can we more tightly scope this?

Reply:
eslint doesn't work very well here because it doesn't look like we have types for these packages and it doesn't seem to recognize that this file's package.json is the one in the file...so there are a lot of eslint errors that can't be followed up on :(